docs/landing-page.md

```diff
@@ -0,0 +1,110 @@
+# Landing page
+
+_This feature is an experimental feature. The setup and configuration of the landing page needs manual code editing. There's no user interface in Admin panel to configure landing page._
+
+This guide will help you to setup and configure landing page. When the landing page is enabled, it will be served to your users from the root URL of your marketplace. The old root search view will be found from `/s` path.
+
+## Enabling landing page
+
+To enable the landing page, change the `clp_static_enabled` to `true`
+
+```yaml
+# config.yml
+
+clp_static_enabled: true
+```
+
+Restart the server and go to the marketplace root URL. You should now see the landing page with the example content.
```

Comment:
Actually it does work. The example content has listings and categories section defined, but the sections are not referred from the `composition` section which constructs the page.